extractors/personal_identifiers/address_extraction/config.py

```diff
@@ -18,6 +18,9 @@ def get_config():
             "personal_identifiers",
         ],  # first entry should be parent directory
         # bricks integrator information
+        cognition_init_mapping={
+            "@@LABEL@@": "Address"
```

Comment:
Regarding to notion, the label needs to be `Location` not `Address`

Reply:
@FelixKirschKern We have a separate brick to extract location, see #371 
The address extraction is not working properly and is currently set to draft/ not public.